src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,26 @@
+import * as S from './styles'
+
+export type AvatarProps = {
+  image?: string
+  borderColor?: 'white' | 'primary' | 'secondary'
+  borderWeight?: 'normal' | 'thick'
+  size?: 'small'
+}
+
+const Avatar = ({
+  image,
+  borderColor = 'primary',
+  borderWeight = 'normal',
+  size = 'small'
+}: AvatarProps) => (
+  <S.Avatar
+    src={image ? image : '/img/user.jpg'}
```

Comment:
Uma dica é trabalhar com valores default, aí evita ternário.

```suggestion
const Avatar = ({
  image = '/img/user.jpg',
  borderColor = 'primary',
  borderWeight = 'normal',
  size = 'small'
}: AvatarProps) => (
  <S.Avatar
    src={image}
```